pkgs/top-level/cuda-packages.nix

```diff
@@ -38,17 +38,45 @@ let
     versions
     ;
   # Backbone
-  gpus = builtins.import ../development/cuda-modules/gpus.nix;
-  nvccCompatibilities = builtins.import ../development/cuda-modules/nvcc-compatibilities.nix;
-  flags = callPackage ../development/cuda-modules/flags.nix { inherit cudaVersion gpus; };
+  # As values created outside of the package set fixed-point, the presence of attributes (and their names) can rely on
+  # these.
+  # By that same proprety, changes to these values will not be reflected in the package set, since they exist outside
+  # of the fixed-point.
+  fixups = import ../development/cuda-modules/fixups { inherit lib; };
+  gpus = import ../development/cuda-modules/gpus.nix;
```

Comment:
> `# these`

I suppose this is a reminder that "Nix is strict in namesets" which is why "it's significant fixups are independent of the fix-point"

> will not be reflected

These values don't depend on the fix-point, but the fix-point does depend on these values.